<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCustomerWindow</class>
 <widget class="QMainWindow" name="AddCustomerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Please enter the information below&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer Entity Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Street Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>City:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>State:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zip:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="StreetLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CityLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="statecomboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>AL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AZ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ME</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NJ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ND</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WY</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="ZipLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>309</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rejectButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
